presto-product-tests/conf/docker/singlenode-kerberos-hdfs-no-impersonation/docker-compose.yml

```diff
@@ -4,3 +4,5 @@ services:
   presto-master:
     volumes:
       - ../../../conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-no-impersonation/hive.properties:/docker/volumes/conf/presto/etc/catalog/hive.properties
+    ports:
+      - '7778:7778'
```

Comment:
This is under "Expose kerberos port for manual development" commit. Consider broadening commit message like
```
Expose security ports for development

This exposes Kerberos and Presto's TLS ports.

This is not related to product tests execution, it is handy when playing
with kerberos manually.
```

also,
Is this the only (base) image where Presto TLS is enabled? I think there is at least some other using 8443 or something.

Reply:
Yes. 8443 is used for LDAP, I added that also. Additionally, I moved 7778 port exposure to kerberos.yaml, that way all the envrionments which are using this will have this port exposed.